src/app/components/code-block/index.html

```diff
@@ -0,0 +1,54 @@
+<sky-demo-page
+  pageTitle="Code block"
+  skyuxModule="SkyCodeBlockModule"
+  packageName="lib-code-block"
+>
+
+  <sky-demo-page-summary>
+    <p>
+      The code block component formats a block of text as a code snippet with syntax highlighting. The code block also has a header that displays a copy button and, if defined, the language type. To create a code snippet, wrap <stache-code>sky-code-block</stache-code> tags around the code block.
+    </p>
+    <p>
+      If a code block has text within <stache-code>&lt;</stache-code> and <stache-code>&gt;</stache-code> characters, then the component automatically appends a closing tag unless the code snippet is valid HTML. To prevent this, you must use "&lt;" to escape <stache-code>&lt;</stache-code> and "&gt;" to escape <stache-code>&gt;</stache-code>. For example, you must escape the <stache-code>&lt;</stache-code> and <stache-code>&gt;</stache-code> characters in the code snippet <stache-code>public Task&lt;Generic> MethodName()</stache-code> or else the component automatically appends <stache-code>&lt;/Generic></stache-code> to the end: <stache-code>public Task&lt;Generic> MethodName()&lt;/Generic></stache-code>.
+    </p>
+    <p>
+    Using the list below, see if your code snippet language is supported. The languages supported appear in pairs. The first part lists the language and the second part displays the text you use for the <stache-code>languageType</stache-code> property. For example, if your code block is a JavaScript code snippet, use <stache-code>languageType="javascript"</stache-code>.
+  </p>
+  </sky-demo-page-summary>
```

Comment:
We need a note here like in the summary action bar docs (https://developer.blackbaud.com/skyux/components/summary-actionbar) to document how to set up the component. I think it should be something like:

"To use the code block component, you must set it up as a dependency in your SKY UX SPA. First, install the package as part of your SPA dependencies: `npm install @blackbaud/skyux-lib-code-block --save`. Then install its plugin: `npm install @blackbaud/skyux-builder-plugin-code-block --save`. Next, add `SkyCodeBlockModule` as both an import and export in your `app-extras.module.ts` file. Finally, add the code block plugin to your `skyuxconfig.json` file `"plugins": ["@blackbaud/skyux-builder-plugin-code-block"],`." ... (Note: I wrote this quickly, adapting the instructions in the Stache docs site to the format in the summary action bar docs. Feel free to adjust as necessary, and we probably need to run this by Jones and/or Tim to make sure nothing is different for a SKY UX SPA."

Reply:
✔️